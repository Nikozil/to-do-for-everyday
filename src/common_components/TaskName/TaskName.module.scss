@import '../../App.scss';

.taskName {
  word-break: break-all;
  margin-left: 10px;
  margin-bottom: 4px;
}
