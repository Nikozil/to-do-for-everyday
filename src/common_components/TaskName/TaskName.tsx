import React, { ReactNode } from 'react';
import styles from './TaskName.module.scss';

const TaskName: React.FC<PropsType> = ({ children }) => {
  return <span className={styles['taskName']}>{children}</span>;
};

export default TaskName;

interface PropsType {
  children: ReactNode;
}
