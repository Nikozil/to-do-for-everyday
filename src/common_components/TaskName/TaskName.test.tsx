import { render } from '../../utils/tests/test-utils';
import TaskName from './TaskName';

describe('TaskName tests', () => {
  it('rendering and click delete', async () => {
    const { asFragment } = render(<TaskName>Task</TaskName>);

    expect(asFragment()).toMatchSnapshot();
  });
});
