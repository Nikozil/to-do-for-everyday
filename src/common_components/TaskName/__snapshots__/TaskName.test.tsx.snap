// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskName tests rendering and click delete 1`] = `
<DocumentFragment>
  <span
    class="taskName"
  >
    Task
  </span>
   
</DocumentFragment>
`;
